src/setupTests.js

```diff
@@ -23,46 +25,36 @@ const mockedStore = (initial = {}) =>
   ])(initial);
 
 // Use this to test mounted components w/ store connection
-global.mountWithProvider = children => initialState => {
-  const store = mockedStore(initialState);
+global.mountWithProviders = children => initialState => {
+  const store = mockedStore({
```

Comment:
We can move this `store` constant and simply call `store` from `mountWithProviders` and `shallowWithProviders`.